### PR TITLE
Fix opa_cli and opa_api resource unable to verify empty result {}

### LINKED ISSUE #5672: opa_cli and opa_api resources unable to verify empty result {}
# Describe the problem

`opa_cli` and `opa_api` are unable to verify empty query results as the output is the empty hash.

### PR DESCRIPTION
**Pull request by Vasu1105**
Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

## Description

`opa_cli ` and `opa_api` get the output in the JSON format and when the output is empty there is no resource property that is able to check this empty result. 

Introduced `result` property to verify empty query results.
## Related Issue

Fix #5672
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs-chef-io/content/inspec/resources/opa_api.md [hunk 1]
```diff
@@ -62,6 +63,12 @@ For a full list of available matchers, please visit our [matchers page](/inspec/
 
 ## Properties
 
+### result
+
+The `result` property checks whether query output is nil.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/opa_api.md]:
```suggestion
The `result` property checks whether the resource query returns an empty result.
```

File: docs-chef-io/content/inspec/resources/opa_cli.md [hunk 2]
```diff
@@ -71,6 +72,12 @@ For a full list of available matchers, please visit our [matchers page](/inspec/
 
 ## Properties
 
+### result
+
+The `result` property checks whether query output is nil.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/opa_cli.md]:
```suggestion
The `result` property checks whether the resource query returns an empty result.
```

File: docs-chef-io/content/inspec/resources/opa_api.md [hunk 3]
```diff
@@ -50,6 +50,7 @@ An OPA query as a JSON data file or a string in JSON format.
 The following examples show how to use this Chef InSpec audit resource.
 
     describe opa_api(url: "localhost:8181/v1/data/example/allow", data: "input.json") do
+      its["result"] { shoule_not be nil }
```
Comment (damacus) [inline docs-chef-io/content/inspec/resources/opa_api.md]:
```suggestion
      its(["result"]) { should_not be nil }
```

We should probably be consistent with parenthesis here.

**Comment by clintoncwolfe** [comment]
This is failing the linting check. @Vasu1105, could you update?

**Comment by Vasu1105** [comment]
> This is failing the linting check. @Vasu1105, could you update?

Thanks, @clintoncwolfe. Did the required changes.

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

**Pull request merged**